StockServiceFactory/src/main/java/edu/mbrooks/advanced/services/WebStockService.java

```diff
@@ -0,0 +1,25 @@
+package edu.mbrooks.advanced.services;
+
+import edu.mbrooks.advanced.model.Quotes;
+
+/**
+ * This API describes how to get stock data from an external resource.
+ */
+public interface WebStockService {
```

Comment:
probably not the best name. why didn't use yahoo?

Reply:
I didn't use Yahoo, because I had everything coded before we got a good API URL - the one in the notes didn't function.